code/modules/research/designs/bluespace_designs.dm

```diff
@@ -3,7 +3,10 @@
 ///////////////Bluespace/////////////////
 /////////////////////////////////////////
 
-/datum/design/beacon
+/datum/design/bluespace
+	autolathe_exportable = FALSE
+
```

Comment:
I don't like this pattern, why do it like this?

Reply:
Less copypaste than just putting the var under every design, but if you don't like it I can do it the normal way

Reply:
I would prefer the normal way. it amounts to just making a /not_exported typepath, clutters the typetree